io.asgardio.tomcat.saml.agent.sample/src/main/resources/sample-app.properties

```diff
@@ -28,6 +28,8 @@ SAML2.SPEntityId=sample-app
 
 IndexPage=/sample-app/index.html
 
+ErrorPage=/error.jsp
```

Comment:
do we need '/'

Reply:
Yes. If we are not adding it here, we'd have to do it in code then.